src/test/java/unit/CartTest.java

```diff
@@ -0,0 +1,43 @@
+package unit;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+public class CartTest {
+    private static Cart cart;
+    private static final double TAX = 0.2;
+
+    @BeforeAll
```

Comment:
According requirements you should also use 'After' annotation. Please add it (for example to remove test files or so)